s3-cloudfront-spa/README.md

```diff
@@ -0,0 +1,40 @@
+# S3 Hosted Website Served by a CloudFront Distribution restricted by Cloudfront Origin Access Control (OAC)
+
+This repo contains serverless patterns showing how to setup a S3 website hosting bucket that is served by a CloudFront distribution that also obfuscates the CloudFront Distribution domain via Cloudfront Origin Access Control (OAC).
+
+![Demo Project Solution Architecture Diagram](diagram.PNG)
+
+- Learn more about these patterns at https://serverlessland.com/patterns.
+- To learn more about submitting a pattern, read the [publishing guidelines page](https://github.com/aws-samples/serverless-patterns/blob/main/PUBLISHING.md).
+
+Important: this application uses various AWS services and there are costs associated with these services after the Free Tier usage - please see the [AWS Pricing page](https://aws.amazon.com/pricing/) for details. You are responsible for any AWS costs incurred. No warranty is implied in this example.
+
+## Requirements
+
+* AWS Account
+* AWS CLI already configured with Administrator permission
+* Terraform
+* Clone this repo!
+
```

Comment:
Please use the format from the template: https://github.com/aws-samples/serverless-patterns/tree/main/_pattern-model

Reply:
updated and applied the format